src/OrchardCore.Modules/OrchardCore.Tenants/Services/TenantShapeTableProvider.cs

```diff
@@ -1,10 +1,12 @@
 using OrchardCore.DisplayManagement;
 using OrchardCore.DisplayManagement.Descriptors;
 using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Modules;
 using OrchardCore.Tenants.ViewModels;
 
 namespace OrchardCore.Tenants.Services;
 
+[Feature("OrchardCore.Tenants")]
```

Comment:
Idem, I think not needed when related to the main feature of the module.

Reply:
done